src/pocketmine/block/Block.php

```diff
@@ -279,6 +279,37 @@ public static function init(){
 			}
 		}
 	}
+	
+	/**
+	 * @param int $blockID
+	 *
+	 * @return int 0-15
+	*/
+	public static function getSkyLightResistance($blockID){
```

Comment:
... why didn't you use `Block::$lightFilter`? It has correct values already.

Also bear in mind that blocks don't specifically have a _sky_ light filter, just a light filter overall. Read http://minecraft.gamepedia.com/Light#Light-filtering_blocks